### DISCUSSION TITLE
Install this connection https://opendds.readthedocs.io/en/latest-release/devguide/quickstart/docker.html After completing all the steps, how does Docker Opendds interact with Java code

### DISCUSSION THREAD

**Discussion by libing331** [OpenDDS/OpenDDS#4517] [General]
Install this connection https://opendds.readthedocs.io/en/latest-release/devguide/quickstart/docker.html After completing all the steps, how does Docker Opendds interact with Java code

**Comment by jrw972** [comment]
The Docker image uses the C++ bindings of OpenDDS.  If you want to use Java, see https://opendds.readthedocs.io/en/latest-release/devguide/java_bindings.html#java
